doc/source/building/build_in_buildservice.rst

```diff
@@ -195,3 +216,43 @@ setting is not sufficient. The following additional line is required:
 .. code:: bash
 
    Release: <CI_CNT>.<B_CNT>
+
+If you are building Vagrant images (see :ref:`setup_vagrant`) then also add
+the repository-type `vagrant`. OBS will then create a `boxes/` subdirectory
+in your download repositories, which will contain JSON files for Vagrant
+[#f4]_.
+
+
+If you have added your repositories to :file:`config.xml`, you will
+probably see a lot of errors of the following type:
+
+.. code:: bash
+
+   unresolvable: have choice for SOMEPACKAGE: SOMEPAKAGE_1 SOMEPACKAGE_2
+
+These can be worked around by adding a lot of ``Prefer:``, ``Substitute:``,
+``Preinstall:`` and ``Support:`` directives. If you do not want to put your
+repository configuration into OBS, then we recommend to initially copy the
+current project configuration of the testing project
+`Virtualization:Appliances:Images:Testing_$ARCH` into your own project as a
+start and tweak it from there instead of starting from scratch.
```

Comment:
The sentence is a bit too long. Also the last sentence should be moved at the first place.

```
Instead of starting from scratch, we recommend to copy the current project
configuration of the testing project `Virtualization:Appliances:Images:Testing_$ARCH`
into your own project. 
<<Maybe add some benefits here>>
Modify/Adapt your OBS repository configuration.
```